README.md

```diff
@@ -15,6 +15,7 @@ Modify the provided Vue 3 app to match [this mock](https://www.figma.com/file/sw
 ### Links
 
 - Figma Mock: <https://www.figma.com/file/swzJVL624G434CVdWi3FLv/Core-UI-Team-Project>
+- Acceptance criterias: <https://docs.google.com/document/d/1AIXTtrEMZBnfoLYXDlBYiEB-BTk7XNt2QlY7jWYdPv0/edit?tab=t.0#heading=h.8hapmwf98sj>
```

Comment:
Can we just put this info here in the readme instead of referencing another document?

Reply:
Why can't we reference the doc? The acceptance criteria are spelled out much more clearly in the doc and that way we don't have to worry about maintaining in two places (we can't get rid of the doc because we will always need something to work off of before we expose the take home).